rust/rust.dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM janitortechnology/ubuntu-dev
+
+# No additional dependencies are required.
+
+# Download Rust's source code.
+RUN git clone https://github.com/rust-lang/rust /home/user/rust
+WORKDIR /home/user/rust
+
+# Configure Cloud9 to use Rust's source directory as workspace (-w).
+RUN sudo sed -i "s/-w \/home\/user/-w \/home\/user\/rust/" /etc/supervisord.conf
+
+# Add build configuration preset
+COPY config.toml .
+
+# Build Rust.
+RUN ./x.py build
+
+# Configure Janitor for Rust
+ADD janitor.json /home/user/
```

Comment:
Nit: Let's start using `COPY` instead of `ADD` (I know all other Dockerfiles till have `ADD`, but let's progressively migrate to `COPY` where `ADD` is not needed).